jfix-stdlib-concurrency/src/main/java/ru/fix/stdlib/concurrency/threads/ReschedulableScheduler.java

```diff
@@ -121,8 +121,10 @@ public void run() {
             // previously launched task still working,
             // schedule changed and triggered new scheduled task to launch
             if(!taskIsRunning.compareAndSet(false, true)){
+                log.trace("Preventing concurrent task launch; scheduledFuture={}", scheduledFuture);
```

Comment:
Also print System.identityHashCode(scheduledFuture)  not only scheduledFuture.toString